config/index.ts

```diff
@@ -81,13 +81,32 @@ export function writeConfig(options: WriteConfigOptions = {}): void {
   }
 }
 
+export function getHiddenOrDeprecatedConfigPath(
```

Comment:
Do we need this? Is it any different from what `getConfigPath()` is doing? What if we defined getConfigPath as `getConfigPath(path?: string, useHiddenConfig?: boolean)`. Every util in this file works for both the deprecated and hidden config, so including the `getHiddenOrDeprecated` part in the name isn't necessary.

Reply:
In my original pass, I deleted the path argument from `getConfigPath` function in this file, and @camden11 pointed out that that would be a breaking change. I created a separate function to avoid that, but this is the simpler solution 🤦‍♀️

Reply:
Addressed in https://github.com/HubSpot/hubspot-local-dev-lib/pull/172/commits/e50aca5114cdd58c397440f70c645a25cdc59857